deploy-manage/monitor/autoops/search-ai-lake-view-autoops-serverless.md

```diff
@@ -0,0 +1,62 @@
+---
+applies_to:
+  serverless:
+navigation_title: Search AI Lake view
+---
+
+# Search AI Lake view in AutoOps for Serverless
+
+The Search AI Lake view in AutoOps for Serverless provides visibility into the key [storage billing dimensions](/deploy-manage/monitor/autoops/autoops-for-serverless.md#storage-billing-dimensions) that drive the costs of your serverless projects. This view helps you understand how ingest and storage activities contribute to your storage usage costs with both high-level summaries and detailed index-level and data stream-levels breakdowns.
+
+The Search AI Lake view is available in all regions across AWS, Azure, and GCP.
+
+To get to this view, [access AutoOps](/deploy-manage/monitor/autoops/access-autoops-for-serverless.md) in your project and then select **Search AI Lake** from the navigation menu.
+
+## Project-level insights
+
+{applies_to}`observability:` {applies_to}`security:` On the **Search AI Lake** page, the top half of the screen offers project-level insights into the ingest rate and storage retained usage metrics over a selected time period.
+
+:::{image} /deploy-manage/images/search-ai-lake-project-level-features.png
+:screenshot:
+:alt: Screenshot showing the features in the top half of the Search AI Lake page
+:::
+
+Use the following features to explore this view:
+* Use the built-in **project picker** to switch between projects. This allows you to make quick context changes without needing to navigate back to your {{ecloud}} home page to select a different project.
+* Select **custom time windows** let to explore usage data ranging from the last 3 hours to the last 10 days. The charted data is bucketed per day except when you select a period of up to 24 hours, when it is bucketed per hour.
```

Comment:
* Select **custom time windows** ~let~ to explore usage data ~ranging from the last 3 hours~up to the last 10 days. The charted data is bucketed per day except when you select a period of up to ~24~72 hours, when it is bucketed per hour.